docs/configuration.md

```diff
@@ -18,6 +18,7 @@
 | `barong_jwt_expire_time` | '3600' | integer representation of seconds  | general purpose tokens (reset password, confirm email) expiration time |
 | `crc32_salt` | - | any string value | salt for crc32 algorithm which used to searching in encrypted fields |
 | `api_data_masking_enabled` | true | `true`, `false` | while turned on (`true`) user API will be with ecnrypted user data |
+|`first_superadmin_registration`| false | `false` `true` | while turned on (`true`) first registered user on a platform will be superadmin without any email confirmation |
```

Comment:
"when turned on" sounds more correct than "while turned on"
You can change the line before too